internal/sys/types.go

```diff
@@ -72,6 +74,7 @@ const (
 	BPF_PROG_ATTACH                 Cmd = 8
 	BPF_PROG_DETACH                 Cmd = 9
 	BPF_PROG_TEST_RUN               Cmd = 10
+	BPF_PROG_RUN                    Cmd = 10
```

Comment:
Did you update `btf/testdata/vmlinux-btf.gz` to get the new definitions? If yes that file should be checked in as well.

Reply:
I switched to testdata/vmlinux-btf.gz to generate types.go